pyproject.toml

```diff
@@ -23,6 +23,11 @@ dependencies = [
     "ipykernel",
     "loguru",
     "matplotlib",
+    "mike",
+    "mkdocs",
+    "mkdocstrings[python]",
+    "mkdocs-material",
+    "mkdocs-jupyter",
```

Comment:
Rather use optional dependencies under an optional group called "docs" or "dev". Users of the package won't need to build the docs themselves.